src/share/cxx/Config.hpp

```diff
@@ -26,12 +26,18 @@
 # define HOMMEXX_MPI_ON_DEVICE 1
 #endif
 
+#include <Kokkos_Core.hpp>
+
 #ifdef KOKKOS_ENABLE_CUDA
 # ifndef HOMMEXX_CUDA_MIN_WARP_PER_TEAM
 #  define HOMMEXX_CUDA_MIN_WARP_PER_TEAM 8
 # endif
+# ifndef HOMMEXX_CUDA_MAX_WARP_PER_TEAM
+#  define HOMMEXX_CUDA_MAX_WARP_PER_TEAM 16
```

Comment:
Feel free to set this to 8 rather than 16 if it makes the CAM integration simpler.

Reply:
That issue seems to have disappeared since I merged it into master